concent-vm/install-concent.yml

```diff
@@ -0,0 +1,99 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../containers/versions.yml
+    - ../concent-builder/repositories.yml
+    - consts.yml
+  tasks:
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:   Clone concent repository
+          git:
+            repo:    "{{ repositories.concent.url }}"
+            dest:    "{{ concent_dir }}"
+            version: "{{ concent_version }}"
+            clone:   yes
+            update:  yes
+
+        - name:   Create virtualenv for concent
+          command:  virtualenv ~/virtualenv --python python3.6 creates="~/virtualenv"
+
+        - name:   Install requirements for concent
+          pip:
+            chdir:              "{{ concent_dir }}/concent_api/"
+            requirements:       "{{ item }}"
+            virtualenv:         ~/virtualenv
+            virtualenv_python:  python3.6
+          with_items:
+            - requirements.lock
+            - ../requirements-development.txt
+
+        - name:    Install middleman and signing-service
+          command: "../../virtualenv/bin/python3.6 setup.py develop"
+          args:
+            chdir:   "{{ concent_dir }}/{{ item }}"
+          with_items:
+            - middleman_protocol
+            - signing_service
+
+        - name:   Install gunicorn
+          pip:
+            name:        gunicorn
+            virtualenv:  ~/virtualenv
+            version:     "{{ gunicorn_version }}"
+
+    - become:      yes
+      become_user: root
+      block:
+        - name:   Copy extra_settings.py from host computer to the remote server
+          copy:
+            src:    "{{ item }}"
+            dest:   "{{ concent_dir }}/concent_api/concent_api/settings/{{ item }}"
+            owner:  vagrant
+            group:  vagrant
+            mode:   0644
+          with_items:
+            - extra_settings.py
+            - local_settings.py
```

Comment:
Only `extra_settings.py` was supposed to be copied to the machine.
`local_settings.py` should be created from a template.

Reply:
Oh, nevermind. I see that this file is actually stored in the repository.

The task `name` is just confusing because it suggests that this part is supposed to upload user's own settings. I think these two uploads should be in separate steps with descriptions that make things clear.